src/Main.java

```diff
@@ -1,5 +1,37 @@
 public class Main {
+    private static TicTacToeGUI gui;
+    private static TicTacToeGame game;
+
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        game = new TicTacToeGame();
+        gui = new TicTacToeGUI();
+
+        gui.onNewGame(Main::start);
+        gui.onMove(Main::move);
+
+        start();
     }
-}
\ No newline at end of file
+
+    private static void start() {
+        game = new TicTacToeGame();
```

Comment:
Game instantiation is clear; however, consider adding error handling for invalid inputs.